tasks/team-access.yaml

```diff
@@ -3,85 +3,93 @@
 
 - include_vars: vars/acs-team-access-config.yaml
 
-# Create Openshift Authenticator - created as part of CCM
+- include_tasks: create_api_token.yaml
 
-# Get OpenShift Authenticator to retrieve the ID
-- name: Get Auth Provider Id
-  uri:
-    url: "https://{{ api_endpoint }}/v1/authProviders?name=Openshift"
-    method: GET
-    validate_certs: no
-    headers:
-      authorization: "{{ authorization }}"
-    body_format: json
-    status_code: 200
-  register: authprovider_result
+- name: Start to create components, if failed, revoke the API token anyway
+  block:
+    # Create the access scopes
+    - name: Create Access Scopes
+      uri:
+        url: "https://{{ api_endpoint }}/v1/simpleaccessscopes"
+        method: POST
+        validate_certs: no
+        headers:
+          authorization: "{{ authorization }}"
+        body_format: json
+        body: "{{ item }}"
+        status_code: [200, 409]
+      with_items: "{{ access_scopes }}"
 
-# Create the permission sets - created as part of CCM
+    # Need to get the permission sets to get IDs for Roles
+    - name: Get Permission Sets
```

Comment:
why query the permission set if you are not doing anything with it? instead you should check if `permission_set: Namespace Admin` exists.

Reply:
This is been used at `create_role.yaml` in create role step